src/main/resources/jenkins/plugins/collabnet/steps/PublishWebhookStep/help-credentialsId.html

```diff
@@ -15,5 +15,5 @@
   -->
```

Comment:
Is this file renamed/moved from EventQ step? If so, create a new file for Webhook step..

Reply:
oh moved it instead of copying it. will add a new file.